src/videojs-http-streaming.js

```diff
@@ -64,16 +64,16 @@ const Hls = {
   'BUFFER_LOW_WATER_LINE_RATE',
   'BANDWIDTH_VARIANCE'
 ].forEach((prop) => {
-  Object.defineProperty(Hls, prop, {
+  Object.defineProperty(Vhs, prop, {
```

Comment:
We may need to keep HLS around for these too, as they are pseudo options unfortunately.